include/wx/charts/wxmath2dplotctrl.h

```diff
@@ -72,6 +72,9 @@ class wxMath2DPlotCtrl : public wxChartCtrl
 private:
     wxMath2DPlot m_math2dPlot;
     wxMenu m_contextMenu;
+    wxMenu *m_subMenu;
```

Comment:
If instead pointer used just wxMenu member - got an runtime error for unknown reasons to me.